dividers/src/main/java/com/karumi/dividers/Sublayer.java

```diff
@@ -40,7 +40,11 @@ public boolean isCloserThan(Sublayer sublayer) {
   }
 
   public Drawable getDrawable() {
```

Comment:
We should either perform the mutation at the latest moment possible (that is, right before calling setBounds) or mutate input drawables in DividerBuilder so that we don't need to think about it later on. I personally prefer the latter to avoid similar bugs in the future.